docs/maui/views/RatingView.md

```diff
@@ -0,0 +1,450 @@
+			ItemShapeSize = 37,
+			MaximumRating = 5,
+			RatingFill = RatingFillElement.Shape,
+			Rating = 4.5,
+			ItemShape = RatingViewShape.Star,
+			ShapeBorderColor = Colors.Red,
+			ShapeBorderThickness = 1,
+			Spacing = 3,
+		};
+
+		Content = ratingView;
+	}
+}
+```
+
+## Properties
+
+| Property | Type | Description |
+|---|---|---|
+| BackgroundColor | `color` | Gets or sets the Color which will fill the background of an element. This is a bindable property. |
+| BindingContext | `object` | Gets or sets an object that contains the properties that will be targeted by the bound properties that belong to this BindableObject. This is a bindable property. |
+| CustomItemShape | `string` | Gets or sets the `Path` for custom rating item shapes. This is a bindable property. |
+| EmptyColor | `Color` | Gets or sets the color that is applied to the unfilled (empty) rating items.  The default value is Transparent.  This is a bindable property. |
+| FilledColor | `Color` | Gets or sets the Color that is applied to the filled (rated) portion of each rating item.  The default value is Yellow. This is a bindable property. |
+| IsReadOnly | `bool` | Gets whether this layout is readonly. The default value is false.  This is a bindable property. |
+| ItemShape | `RatingViewShape` | Gets or sets the rating item shape.  The property is of type [`RatingViewShape`](#set-item-shape) and is an enumeration. The default value is Star.  This is a bindable property. |
+| ItemShapeSize | `double` | Gets or sets the size of the rating item shape.  The default value is 20. |
+| MaximumRating | `byte` | Gets or sets the maximum number of ratings. The range of this value is 1 to 25; values outside this range will be set to the nearest valid value.  The default value is 5. This is a bindable property. |
+| RatingChanged | `EventHandler<RatingChangedEventArgs>` | Event occurs when the rating is changed. |
+| RatingFill | `RatingFillElement` | Gets or sets a value indicating how the fill is applied against the entire rating item or just the shape. The property is of type [`RatingFillElement`](#set-rating-fill) and is an enumeration. The default value of this property is Shape.  This is a bindable property. |
+| Rating | `double` | Gets or sets a value indicating the current rating value, allowing for both pre-defined ratings (e.g., from previous user input or stored data) and updates during runtime as the user interacts with the control.  The default value is 0.  This is a bindable property. |
+| ShapeBorderColor | `Color` | Gets or sets the border color of the rating item shape. The default value of this is Grey.  This is a bindable property. |
+| ShapeBorderThickness | `Thickness` | Gets or sets the border thickness of the rating item shape.  The default value is a Thickness with all values set to 1.  This is a bindable property. |
+| Spacing | `double` | Gets or sets the spacing between rating item elements.  The default value is 10.  This is a bindable property. |
```

Comment:
If we are inheriting from HorizontalStackLayout as the next section suggests then we don't need to include Spacing here